libsel4/sel4_arch_include/x86_64/interfaces/sel4arch.xml

```diff
@@ -28,7 +28,24 @@
         <member name="fs_base"/>
         <member name="gs_base"/>
     </struct>
-
+    <struct name="seL4_VCPUContext">
+        <!-- eax -> ebp are actually rax -> rbp. Kept for consistency -->
+        <member name="eax"/>
```

Comment:
eax-ebp are well-known names for 32-bit registers. should switch to rax-rbp instead.